pnpm-lock.yaml

```diff
@@ -189,7 +189,7 @@ importers:
       '@sveltejs/adapter-cloudflare-workers': link:../../../adapter-cloudflare-workers
       '@sveltejs/adapter-netlify': link:../../../adapter-netlify
       '@sveltejs/adapter-vercel': link:../../../adapter-vercel
-      '@sveltejs/kit': 1.0.0-next.144_svelte@3.40.0
+      '@sveltejs/kit': link:../../../kit
```

Comment:
a documentation PR shouldn't update the lockfile

Reply:
Fixed